relay/sources/relay-cleanup.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# remove all files and their metadata in BASEDIR that have expired
+
+BASEDIR="/var/rudder/shared-files"
+date=$(date +%s)
+find "${BASEDIR}" -type f -name '*.metadata' | xargs grep -H 'expires=' | sed 's/^\(.*\).metadata:expires=/\1 /' |
```

Comment:
is the expiration an integer timestamp ?

Reply:
This is the previous cronjob without modification. (But yes it is a timestamp IIRC)